themes/root/templates/Helpers/icons/unicode.phtml

```diff
@@ -0,0 +1,5 @@
+<?php
+    $this->headLink()->appendStylesheet($this->src);
+    $className = trim('icon icon--font icon--unicode ' . ($this->extra['class'] ?? ''));
+?>
+<span class="<?=$className?>"<?=$this->attrs ?> role="img" aria-hidden="true">&#x<?=$this->escapeHtml($this->icon)?>;</span>
```

Comment:
As per #3084, we should probably escape `$className` here for consistency.

Reply:
Should be all sorted now.